libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -87,9 +87,10 @@ export class Bucket extends Resource {
   /**
    * Creates a topic for subscribing to notification events
    * @param actionType
-   * @returns the created topi
+   * @returns the created topic
+   * @internal
    */
-  protected createTopic(actionType: BucketEventType): Topic {
+  protected _createTopic(actionType: BucketEventType): Topic {
```

Comment:
I have to mark them somehow so they'll be skipped here:
https://github.com/winglang/wing/blob/d05234b82dd30a443bab07bcb87773c89a2a407b/libs/compatibility-spy/src/index.ts#L51-L56
what do you think will be a better approach?

Reply:
If you can correlate the class with it's data in the .jsii file, it may be better to use that as the source of truth since it only has public stuff (due to the `@internal`).

In fact, it may be better to start from the `.jsii` and iterate over it to create the proxies

Reply:
Where can I find it?

Reply:
The .jsii file is at the root of a JSII package. So for the SDK it'll be at `libs/wingsdk/.jsii` (only available once compiler or packaged)

In our repo, you can check out this file for some examples of traversing it (using the jsii-reflect package) https://github.com/winglang/wing/blob/main/apps/wing-api-checker/src/index.ts